components/signup/signupform.tsx

```diff
@@ -259,20 +279,74 @@ export default function SignUpForm() {
 }
 
 
-const PasswordStrengthBar = ({ passwordreset }: {
-    passwordreset: number
-}) => {
+// const PasswordStrengthBar = ({ passwordreset }: {
+//     passwordreset: number
+// }) => {
 
 
-    return (
-        <div className="flex flex-row justify-start gap-2">
-            <div
-                className={
-                    `h-2 w-full rounded-full ${passwordreset >= 1 ? "bg-green-500" : "bg-gray-300"}`
-                }
-            >
+//     return (
```

Comment:
@pratyushsingha can you please clean the lines which are comment out.

Reply:
ok